doc/languages-frameworks/android.section.md

```diff
@@ -216,25 +246,22 @@ In addition to prebuilt APKs, you can also bind the APK parameter to a
 
 Querying the available versions of each plugin
 ----------------------------------------------
-When using any of the previously shown functions, it may be a bit inconvenient
-to find out what options are supported, since the Android SDK provides many
-plugins.
+repo.json provides all the options in one file now.
 
-A shell script in the `pkgs/development/mobile/androidenv/` sub directory can be used to retrieve all
+A shell script in the `pkgs/development/mobile/androidenv/` subdirectory can be used to retrieve all
 possible options:
 
 ```bash
-sh ./querypackages.sh packages build-tools
+sh ./querypackages.sh packages
```

Comment:
This should not be called with sh, its a nix-shell.

```suggestion
./querypackages.sh packages
```

Reply:
Thanks, I think this was left over from the old docs.